src/enzo/Grid_ZeusSolver.C

```diff
@@ -161,7 +173,7 @@ int grid::ZeusSolver(float *gamma, int igamfield, int nhy,
   ke = GridEndIndex[2];
 
   //  If NumberOfGhostZones is set to 4, then use the extra space
-
+/*
```

Comment:
is commenting this out going to cause problems with, e.g., cosmology simulations?

Reply:
This was reverted to the original handling.  The QuantumPressure version is really a generalization to handle cases when using more than 4 ghost zones, but maybe this PR is not the place to generalize this.  I have kept the original handling to make sure nothing changes in non FDM version.